test/test_khard.py

```diff
@@ -161,3 +161,14 @@ def test_find_email_addresses_finds_all_emails_with_other_headers_too(
                 username="mjones",
                 domain="machine.example")]
         self.assertEqual(expected, addrs)
+
+class TestSearchContacts(unittest.TestCase):
+
+    def test_get_contact_list_by_user_selection(self):
```

Comment:
When we want to merge this we need to mark it as an expected failure. Use the `unittest.expectedFailure` decorator.